MetaMask/metamask-mobile
# refactor: rename the feature flag since it had a typo

## **Description**
This PR just renames the feature flag since it had a typo, making it difficult to find for someone looking for multichain since mu-l-ti had a typo.


## **Related issues**

Fixes:

## **Manual testing steps**

1. To test without the mutlichain mocked screens, in js.env toggle the feature flag off (the default is also off) by setting `export MM_MULTICHAIN_V1_ENABLED=""`
2. Then connect to a dapp via the in-app browser and see the current screens currently used in prod
3. To enable mulitichain mocked screens, set the `export MM_MULTICHAIN_V1_ENABLED="1"` and connect to a dapp to see new mocked multichain screens


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
